### PR TITLE
feat: Increase tauri runtime threads efficiency

### LINKED ISSUE #57: [APP - RUST] - Increase tauri runtime threads efficiency
In async tauri::command we should avoid using blocking method variants from dependencies. 
Network and s3 modules are already _"async friendly"_ with some minor tweaks still possible.
fs should take adavnatge of [tokio::fs](https://docs.rs/tokio/latest/tokio/fs/index.html) and ftp can have some benefits switching to [AsyncFtpStream](https://docs.rs/suppaftp/5.2.1/suppaftp/type.AsyncFtpStream.html)

### PR DESCRIPTION
**Pull request by newfla**
Closes #57 

tokio and tokio-util dependencies have been introduced in order to have async-io in fs, s3 and ftp modules.

### CODE REVIEW

**Comment by brainrepo** [comment]
Simply awesome, I left just a comment about changing the function name in [packages/app/src-tauri/src/commands/fs.rs](https://github.com/fourviere/fourviere-podcast/pull/58/files/90d661a93157eae0d380ba6c6c8a8af36c9eb247#diff-7934beba9d1a48329c9341e44a36605b04e149a1a03f62337304e3c68f77598e)

**Review by brainrepo — CHANGES_REQUESTED** [review]

File: packages/app/src-tauri/src/commands/fs.rs [hunk 1]
```diff
@@ -1,30 +1,8 @@
-use std::io::prelude::*;
-use std::{fs::File, path::Path};
+use tokio::fs::read_to_string;
 
 #[tauri::command]
 pub async fn read_file(path: &str) -> Result<String, String> {
```
[the remark this answers is not in the document]
Comment (newfla) [reply packages/app/src-tauri/src/commands/fs.rs]: Renamed to read_text_file

File: packages/app/src-tauri/Cargo.toml [hunk 2]
```diff
@@ -19,8 +19,10 @@ reqwest = { version = "0.11", features = ["json"] }
 rust-s3 = "0.33.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-suppaftp = { version = "^5.2.2", features = ["native-tls"] }
```
[the remark this answers is not in the document]
Comment (newfla) [reply packages/app/src-tauri/Cargo.toml:22]: Caret syntax ^x.y.z should be equivalent to =x.y.z

**Review by brainrepo — APPROVED** [review]
GREAT!!!! Thx @newfla !!!!

**Pull request merged**